spring-boot-starter/spring-boot-starter-discovery/src/main/java/io/servicecomb/springboot/starter/serviceregistry/CseServiceRegistryAutoConfiguration.java

```diff
@@ -23,16 +23,14 @@
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
 
-import io.servicecomb.serviceregistry.client.ServiceRegistryClient;
-
 @Configuration
 @ConditionalOnProperty(value = "spring.cloud.service-registry.enabled", matchIfMissing = true)
 @AutoConfigureBefore(ServiceRegistryAutoConfiguration.class)
 public class CseServiceRegistryAutoConfiguration {
```

Comment:
okay once this PR is merged I will re-factor the names and raise another MR immediately, right now the PR code size is big, re-merging/re-basing is adding extra effort.